README.md

```diff
@@ -9,7 +9,7 @@ To use Swiftly in your workflow, specify the repository followed by a tag as wit
 ```yaml
 steps:
   - uses: actions/checkout@v4
-  - uses: vapor/swiftly-action@v0.1
+  - uses: vapor/swiftly-action@v0.2.0
```

Comment:
Ironically, this reminded me that I forgot to push a `v0.2` tag! (And for that matter, the `v0.1` tag was out of date as well.) I would prefer to recommend the `v0.2` tag instead of pinning users to a specific patch version.
```suggestion
  - uses: vapor/swiftly-action@v0.2
```